galaxy_api/api/ui/serializers/namespace.py

```diff
@@ -40,6 +50,27 @@ def update(self, instance, validated_data):
         return instance
 
 
+class NamespaceUpdateSerializer(NamespaceSerializer):
+    '''NamespaceSerializer but read_only 'name'
```

Comment:
1. Please use double quotes for docstrings.
2. For one liner docstrings, please keep the closing """ on the same line.
3. Please put period at the end of the sentence.

Please read official python coding style PEP8 [1]:
https://www.python.org/dev/peps/pep-0008